lib/inspector.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+const assert = require('assert');
+const connect = process.binding('inspector').connect;
+const EventEmitter = require('events');
+
+assert(connect, 'Inspector is not available');
+
+exports.Session = class extends EventEmitter {
+  constructor() {
+    super();
+    this._connection = connect((message) => this._onMessage(message));
```

Comment:
Would prefer using internal `Symbols` for these instead of `_`-prefixed properties.